src/EventListener/DcaListener.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of terminal42/contao-inserttags.
+ *
+ * (c) terminal42
+ *
+ * @license MIT
+ */
+
+namespace Terminal42\InsertTagsBundle\EventListener;
+
+use Contao\Backend;
+use Contao\DataContainer;
+use Contao\Image;
+use Contao\Input;
+use Contao\StringUtil;
+use Doctrine\DBAL\Connection;
+
+class DcaListener
```

Comment:
We also use `@Hook` so we can also use `@Callback` here :)

Reply:
Yeah, I am not a big fan of `@Callback` because then your DCA definition is eventually split between multiple listeners and a single DCA file. It's easier (at least for me) to analyze code when it's all in one place. Let me know if we should change it for this repo.